### DISCUSSION TITLE
Add player arguments for the Sell GUI/Warp/Scavenge menus

### DISCUSSION THREAD

**Discussion by tylerking** [Zrips/CMI#7229] [Feature Requests]
It would be nice to have player arguments for these menus (and any other menus I haven't listed) so we could open the menus via console for the players without having to give the player the command permissions. I have custom menus and NPC's setup and I want them to be the only way a player can access these menus. Thoughts? Thanks in advance.

**Comment by mrfloris** [comment]
I agree, this would be kinda nice to have as an option, that we can open a menu on a player from the console.

Related question; https://github.com/Zrips/CMI/discussions/7225

**Comment by Zrips** [comment]
Next update
